docs-chef-io/content/inspec/cli.md

```diff
@@ -1,17 +1,10 @@
 +++
-title = "InSpec CLI"
-draft = false
-gh_repo = "inspec"
-
-[menu]
-  [menu.inspec]
-    title = "InSpec Executable"
-    identifier = "inspec/reference/cli.md InSpec Executable"
-    parent = "inspec/reference"
-    weight = 10
```

Comment:
How to generate this metadata? Any idea on this?

Reply:
If it helps, this is the menu metadata spec from Hugo https://gohugo.io/content-management/menus/#add-content-to-menus

It can handle yaml, toml, json.